schemaless/sources.py

```diff
@@ -678,6 +684,7 @@ class OEWDPermits(DirectSource):
         'Target Permit Issuance Date': 'target_permit_issuance_date',
     }
     DATA_SF = "https://data.sfgov.org/dataset/Priority-Permits/336t-bzzm"
+    DATA_SF_DOWNLOAD = ""  # TODO: Not public
```

Comment:
I'm following up with Jade about this but they've been busy/MIA, can we still do this if it is not public?

Reply:
Technically, yes, but I'd have to rewrite the code to fetch datasets to authenticate & pull. Hoping they just make it public asap :)

Reply:
Whatever solution we do pursue, I'd like to push it to a followup issue

Reply:
sg, as talked about this won't be public so we'll either have to make it clear we're not using for DA and OCII projects or make ti work with the auth